### DISCUSSION TITLE
Difference from conjure?

### DISCUSSION THREAD

**Discussion by agriffis** [liquidz/elin#2] [Q&A] [answered]
Hello! I'm currently using https://github.com/Olical/conjure and I was wondering how this project is different. Do you know?

**Comment by liquidz** [comment, marked as answer]
@agriffis I didn't have used Conjure, so I cannot compare the functionalities.

But a major outward difference is that Conjure is written in Lua (Fennel) and is a Neovim-specific plugin without any special dependencies. It also supports a wide range of languages beyond Clojure.

On the other hand, Elin is mostly written in Clojure and depends on Babashka. While it works not only in Neovim but also in Vim, currently, it only supports Clojure.

**Reply by agriffis** [reply]
Okay, thank you for the response!
